interpreter/exec/simd.ml

```diff
@@ -112,6 +125,27 @@ struct
   let to_i16x8 = Rep.to_i16x8
   let to_i32x4 = Rep.to_i32x4
 
+  module MakeVec (Int : Int.S) (Convert : sig
```

Comment:
Do we actually need to have this be a functor?

Reply:
Nope, a bit of consistency with the rest, but I can change this to be a plain module :)

Reply:
Changed to a module.